lib/angular-evaporate.js

```diff
@@ -147,6 +157,11 @@
                       (100 - that.$percent) / that.$percent *
                       (now - that.$started) / 1000
                     );
+                    var progress = (arg * 100).toFixed(2);
+                    var delta = (new Date() - that.$started) / 1000;
+                    var loaded = (progress * that.sizeBytes) / 100;
```

Comment:
I'm no expert too, just copied some info from the links above) I guess, you should check `that.file.size` instead.